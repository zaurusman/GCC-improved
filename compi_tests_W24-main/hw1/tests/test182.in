mM0PdHlDo8Y"else"w \\\ \" yo5czGiERk N8k2ZBq Lc\" \" \n kXqLbLGIp \" Y \" \" \r \\  >=break"w \\\ \" yo5czGiERk N8k2ZBq Lc\" \" \n kXqLbLGIp \" Y \" \" \r \\ +void
continue"\x23 uZMgfG7w \r w 017214\tfalsegf5RSaSOYp/(trueLaTdQ6Umiltrue"\C O 36329 XXjUA5k02ApHIvI 95140 s YBOlqATcXJsZ "\\ andtrue*"\n ihvE \t \t pCbIL3DMZgF9HJU 58347 68472 eGdUFJOfjeY \x70 56625 elsewhile"M5zq \" W L \" 76814 \n \r JFnGWPp50089 \x3b \x3F oror
/not"72830 \\\ Yadyky1a2WjmSSG \x72 VMEwUl36 94477 7190 \r \r \n dr7Cb \0 \"\" 20734 \"true"\r J42mwYRkXGtzecBwoF2Pandd2womsqL"\t\" \r "4560 \\ [ 0 \" ykGC+"4560 \\ [ 0 \" ==continue<="XEirp"bXViJQKnHg \0 \R \\ 42248 p 84148 \r V d oNrc 68424 \\\ \n \n \n D \\\ 

>trueand;inteXnotboolLTZnbreak"Y 79612 t4 jdBjOZrNPO2c24M \\ \0 \\ \0 H+not"20638 48492C \" \n \\ \t \\ \\\" \x5D"\\\ h ==;else=
Hl<"\\\ 70322 \x38 \" 41806 bUdA4NTaCwK26 \r Iy4PklzAmtEqpd7 \\ E5XQd0\\ ==*==breakwhileyLx"L\\\ vY5wH \\\ \H \n \n \\\ \\ x D p 
===-<-"X \0 \x32 \n 40779 e8r3Mj \" 46805 \0 61246 ; 50705 \0 \r \xEB G \\ !=notW13LpHdirk<=>2cLq"I \\ 20992 \\ 70441 26878 E 12327 \0 v 55673 KgLMYTwvyKG Q0 26902 \x30 \x6D 84853 h \x63 notwhile=return<=else