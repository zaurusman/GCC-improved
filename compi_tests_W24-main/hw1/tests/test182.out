1 ID mM0PdHlDo8Y
1 STRING else
1 ID w
Error \
